htrace-zipkin/src/main/java/org/apache/htrace/impl/ScribeTransport.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.htrace.impl;
+
+import com.twitter.zipkin.gen.LogEntry;
+import com.twitter.zipkin.gen.Scribe;
+
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.htrace.Transport;
+import org.apache.htrace.core.HTraceConfiguration;
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TBinaryProtocol;
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.transport.TFramedTransport;
+import org.apache.thrift.transport.TSocket;
+import org.apache.thrift.transport.TTransport;
+import org.apache.thrift.transport.TTransportException;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ScribeTransport implements Transport {
+
+  /**
+   * this is used to tell scribe that the entries are for zipkin..
+   */
+  public static final String CATEGORY = "zipkin";
+
+
+  private static final Log LOG = LogFactory.getLog(ScribeTransport.class);
+  /**
+   * Default hostname to fall back on.
+   */
+  private static final String DEFAULT_COLLECTOR_HOSTNAME = "localhost";
+  public static final String DEPRECATED_HOSTNAME_KEY = "zipkin.collector-hostname";
+  public static final String HOSTNAME_KEY = "zipkin.scribe.hostname";
```

Comment:
I would prefer if we drop the zipkin specific terminology of collector (with default port different than scribe's default). How about using port 1463 and referring to these as scribe host and scribe port?

Very soon I want to completely push scribe to the periphery of zipkin where it can die. The port chosen in the first place was an accident of copy paste of port dodging on laptop install into docker. Internally at Twitter we always used scribe's default